kvdb-rocksdb/src/lib.rs

```diff
@@ -755,6 +769,13 @@ impl KeyValueDB for Database {
 	}
 
 	fn io_stats(&self, kind: kvdb::IoStatsKind) -> kvdb::IoStats {
+		let rocksdb_stats = self.get_statistics();
+		let cache_hit_count = rocksdb_stats.get("block.cache.hit").map(|s| s.count).unwrap_or(0u64);
+		let overall_stats = self.stats.overall();
+		let old_cache_hit_count = overall_stats.raw.cache_hit_count;
+
+		self.stats.tally_cache_hit_count(cache_hit_count - old_cache_hit_count);
```

Comment:
this hack is needed because `get_statistics()` returns only `overall` stats, it has probably a race condition
idea how to make this better are welcome!